src/features/unmarkedWsNoticeList/printAllStickersButton/PrintAllStickersButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useTranslation } from 'react-i18next';
+
+import Button from '../../../common/button/Button';
+import { UNMARKED_WINTER_STORAGE_NOTICES_STICKERS_QUERY } from './queries';
+import { CustomerInfo } from '../UnmarkedWsNoticeList';
+import { generateAndSaveStickerPDF } from '../utils';
+import { UNMARKED_WINTER_STORAGE_NOTICES_STICKERS } from './__generated__/UNMARKED_WINTER_STORAGE_NOTICES_STICKERS';
+import styles from './printAllStickersButton.module.scss';
+
+const PrintAllStickersButton = () => {
+  const { t } = useTranslation();
+  const { loading, data } = useQuery<UNMARKED_WINTER_STORAGE_NOTICES_STICKERS>(
+    UNMARKED_WINTER_STORAGE_NOTICES_STICKERS_QUERY
+  );
+
+  if (loading) {
+    return (
+      <Button disabled={loading} className={styles.margin}>
+        {t('unmarkedWsNotices.list.allStickerPrint.loading')}
+      </Button>
+    );
+  }
+
+  const customers: CustomerInfo[] =
+    data?.winterStorageNotices?.edges.reduce<CustomerInfo[]>((acc, edge) => {
+      if (!edge?.node?.lease?.stickerNumber) {
+        return acc;
+      }
+      const { firstName, lastName, address, municipality, zipCode } = edge.node;
+      return [
+        ...acc,
+        {
+          firstName,
+          lastName,
+          address,
+          municipality,
+          zipCode,
+          stickerNumber: edge.node.lease.stickerNumber,
+          stickerSeason: edge.node.lease.stickerSeason,
+        },
+      ];
+    }, []) ?? [];
```

Comment:
Would it be better to extract it to a `utils` module?

Reply:
Done.